src/main/resources/anniv5Resources/localization/eng/cosmoscommandpack/Cardstrings.json

```diff
@@ -21,7 +21,7 @@
   },
   "${ModID}:Recall": {
     "NAME": "Recall",
-    "DESCRIPTION": "Whenever a card is ${ModID}:Amplified, put a card from your Exhaust pile into your hand."
+    "DESCRIPTION": "The first time a card is ${ModID}:Amplified each turn, put a card from your Exhaust pile into your hand."
```

Comment:
Another thought I had given how abusable this power is -- have you considered adding onto this "It gains Purge."?

This mechanic is very abusable since there are some exhausting cards that heal or give you permanent benefits like upgrades (Knighthood) or potions (Alclawmize). Getting those back once is cool. Getting those back an unlimited number of times is abusable. That's why I'm suggesting this as a way to keep the fun and power of this mechanic, but prevent it from being abused by bringing back the same card over and over.

Reply:
As in, the card that was Amplified is Purged? That could be a really cool effect!
Like, you have to sacrifice one card to get another back... I like it

Reply:
Or do you mean the card that was Exhumed... either way, great idea ^^

Reply:
I was thinking the card that was Exhumed. I came up with that because it was the most direct way to prevent abusive interactions, and becuase it makes you have to think a bit more -- "do I get back my best card from Exhaust on this turn, or do I wait until next turn when I might need it more, since I can only get it back once?" Otherwise, if you have something like an Adrenaline (or any other generically very strong card) in the Exhaust pile, you'll just get that back turn after turn after turn.

Having the card that was Amplified purge is another interesting idea. It imposes a different kind of limit, since you'll eventually run out of Amplify cards. Though I still like the idea of having the Exhumed card Purge the most since I think it makes for more strategic and interesting gameplay while also preventing degenerate interactions.

Reply:
Nice, started work on the "card that was Exhumed Purges" version

Reply:
Great. I'll take another pass through the review once you've updated it.

For making the exhumed card purge, I'd recommend a card modifier. I don't know if a `PurgeMod` already exists, but you should be able to adapt something like `ExhaustMod`.